src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicCcm.hpp

```diff
@@ -73,13 +73,14 @@ class CProxy_GlobalCache;
 }  // namespace Parallel
 /// \endcond
```

Comment:
Instead of naming this file with `Ccm`, maybe rename to `EvolveGhAndCharacteristics` so that it's more general and can be both CCE and CCM.